src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -3,6 +3,7 @@ import { IChatMessageProps } from '../../../../components/Chat/ChatMessage'
 import { IChatMessageVariables } from './chatMessageVariables'
 import { pxToRem } from '../../../../lib'
 
+const rem10 = pxToRem(10)
```

Comment:
would rather suggest name like `px10asRem` - otherwise the name is a bit misleading and reads like '10 rem'